packages/page-statistics/src/App.tsx

```diff
@@ -0,0 +1,82 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { randNumber, randRecentDate, randSoonDate } from '@ngneat/falso';
+import { KeygenTable, KeyStatusCard, Logo, ThemeSwitcher } from '@webb-dapp/webb-ui-components';
+import { WebbUIProvider } from '@webb-dapp/webb-ui-components/provider';
+import { FC } from 'react';
+
+const apolloClient = new ApolloClient({
+  cache: new InMemoryCache(),
+  uri: 'http://localhost:4000',
+});
+
+const App: FC = () => {
+  return (
+    <WebbUIProvider>
+      <div className='min-h-[72px] bg-mono-0 dark:bg-mono-180 mb-8 flex justify-between items-center p-4'>
+        <Logo />
+        <ThemeSwitcher />
+      </div>
+
```

Comment:
@AtelyPham can you confirm the spacing between the header bar and content? See image below. It would be good to @monaiuu and @seeddisperser to weigh in. 

<img width="346" alt="Screen Shot 2022-09-15 at 9 49 01 AM" src="https://user-images.githubusercontent.com/29983536/190348775-55893748-2f5c-4aa6-a1fc-e8afa4c220e6.png">

Reply:
Actually, the padding of the content in the design is `32px` at the moment, will discuss more with the design team.

![CleanShot 2022-09-15 at 15 12 11@2x](https://user-images.githubusercontent.com/60747384/190351775-5b83ba13-0af8-405f-a1e6-8fe65dfb95f1.png)